src/main/resources/swagger/rawls.yaml

```diff
@@ -2510,18 +2530,48 @@ paths:
             - openid
             - email
             - profile
-            
+  '/api/user/{userSubjectId}/enable':
```

Comment:
i think we should get answer to the email question about moving these under /admin and just get it done here